src/main/java/com/google/sps/data/UserRepository.java

```diff
@@ -12,4 +12,6 @@
   public Collection<User> getAllUsers();
 
   public String toString();
+
+//   public User getUser(com.google.appengine.api.users.User googleUser);
```

Comment:
should this be removed?

Reply:
Yes. I wasn't sure if I should leave it because Sam will eventually need to implement that method for the PersistentUserRepository, but I can just tell him to add it when he works on it :)

Reply:
Update: I added this back in to the other PR because I needed getUser to be a method for UserRepository. I added a dummy method in the PersistentUserRepository so that it compiles but Sam will fill that in when he works on it next.